treeherder/services/pulse/consumers.py

```diff
@@ -154,6 +153,7 @@ def bindings(self):
     @newrelic.agent.background_task(name='pulse-listener-tasks.on_message', group='Pulse Listener')
     def on_message(self, body, message):
         exchange = message.delivery_info['exchange']
+        print(exchange)
```

Comment:
Don't leave `print` in the code.

Reply:
Sorry. Left it just during the debugging

Reply:
Please remove this. Thanks.